amberdata/README.md

```diff
@@ -11,20 +24,115 @@ Gets the [latest spot VWAP price](https://docs.amberdata.io/reference#spot-price
 
 ```json
 {
- "jobRunID": "1",
- "data": {
-  "status": 200,
-  "title": "OK",
-  "description": "Successful request",
-  "payload": {
-   "timestamp": 1603800660000,
-   "pair": "link_usd",
-   "price": "12.02087667",
-   "volume": "508.31"
+  "jobRunID": "1",
+  "data": {
+    "status": 200,
+    "title": "OK",
+    "description": "Successful request",
+    "payload": {
+      "timestamp": 1603800660000,
+      "pair": "link_usd",
+      "price": "12.02087667",
+      "volume": "508.31"
+    },
+    "result": 12.02087667
   },
-  "result": 12.02087667
- },
- "result": 12.02087667,
- "statusCode": 200
+  "result": 12.02087667,
+  "statusCode": 200
+}
+```
+
+### Balance endpoint
+
+- `dataPath`: Optional path where to find the addresses array, defaults to `addresses`
+- `confirmations`: Optional confirmations param, defaults to `6`
+
+- `addresses`: Addresses to query
+
+  {
+
+  - `address`: Address to query
+  - `coin`: Optional currency to query, defaults to `btc`, one of `(btc|eth|bch|ltc|btsv|zec)`
+  - `chain`: Optional chain to query, defaults to `mainnet`, one of `(mainnet|testnet)` when querying ethereum. Returns Rinkeby data.
+
+  }
+
+```json
+{
+  "id": "1",
+  "data": {
+    "addresses": [
+      {
+        "address": "3EyjZ6CtEZEKyc719NZMyWaJpJG5jsVJL1"
+      },
+      {
+        "address": "38bzm6nhQMFJe71jJw1U7CbgNrVNpkonZF"
+      },
+      {
```

Comment:
Let's make the input example also with two addresses if the output example is for two addresses.